.gitignore

```diff
@@ -52,4 +52,9 @@ dune-project
 ## File-based project format:
 *.iws
 
+# Ligo output
 out/
+
+# Haskell
+haskell/.stack-work/
```

Comment:
I think this line can be removed (it should be covered by the next one already IIRC).

Reply:
Removed.